tests/end-to-end/inplay_market.ts

```diff
@@ -45,16 +48,28 @@ describe("End to end test of", () => {
       assert.equal(e.error.errorCode.code, "MarketEventNotStarted");
     }
 
-    // FIXME could be replaced with a call to market.setEventStartTimestampToNow()
-    while (Math.floor(new Date().getTime() / 1000) <= eventStartTimestamp + 5) {
-      await new Promise((resolve) => setTimeout(resolve, 500));
+    // Move start time up to now
+    await market.updateMarketEventStartTimeToNow();
+    await new Promise((resolve) => setTimeout(resolve, 5000));
```

Comment:
5000? painful

Reply:
We can probably reduce this, I just tidied up the `FIXME`. The `+ 5` was there before. 
```
while (Math.floor(new Date().getTime() / 1000) <= eventStartTimestamp + 5) {
      await new Promise((resolve) => setTimeout(resolve, 500));
```